crypto/doc/api.rst

```diff
@@ -115,6 +115,20 @@ AES - Advanced Encryption Standard APIs
    :project: nrfxlib
    :members:
 
+AES-CBC - AES Cipher Block Chaining Mode
```

Comment:
Most of the sections in the file have APIs in the end, maybe we could add it to all new ones as well for consistency? Except perhaps the ocrypto internal types...

Reply:
Another question is if it should be API and not APIs when only one doxygengroup is presented in the section. But this is relevant for the entire file and does not have to be addressed in this PR.

Reply:
Then this is in the API section anyway so maybe it would be better to just remove APIs/API as it should be obvious

Reply:
True

Reply:
Ok removed all the API(s) in the titles